test/multidimensional_matrix_operators_test.jl

```diff
@@ -0,0 +1,130 @@
+using Test
+using SummationByPartsOperators
+using LinearAlgebra
+using StaticArrays
+using SparseArrays
+
+function SummationByPartsOperators.mass_matrix_boundary(D::SummationByPartsOperators.AbstractDerivativeOperator)
+    T = eltype(D)
+    b = zeros(T, length(grid(D)))
+    b[1] = T(-1.0)
+    b[end] = T(1.0)
+    return Diagonal(b)
+end
+
+function SummationByPartsOperators.integrate_boundary(func, u, D::SummationByPartsOperators.AbstractDerivativeOperator)
+    return func(u[end]) - func(u[1])
+end
```

Comment:
Could it make sense to add functions like these to the source?

Reply:
Yes 👍